composer.json

```diff
@@ -35,7 +35,7 @@
         "cache/integration-tests": "^0.16",
         "predis/predis":           "^1.0",
         "symfony/cache":           "dev-master",
-        "illuminate/cache":        "^5.0"
+        "illuminate/cache":        "^5.4"
```

Comment:
We should skip the PHP 5.5 build for this package as Illuminate 5.4 requires PHP 5.6+. Do you have an idea for this @Nyholm ?